packages/core-components/src/layout/ErrorPage/ErrorPage.tsx

```diff
@@ -62,37 +71,44 @@ const useStyles = makeStyles(
  *
  */
 export function ErrorPage(props: IErrorPageProps) {
-  const { status, statusMessage, additionalInfo, supportUrl } = props;
+  const { status, statusMessage, additionalInfo, supportUrl, stack } = props;
   const classes = useStyles();
   const navigate = useNavigate();
   const support = useSupportConfig();
 
   return (
-    <Grid container spacing={0} className={classes.container}>
-      <Grid item xs={12} sm={8} md={4}>
-        <Typography
-          data-testid="error"
-          variant="body1"
-          className={classes.subtitle}
-        >
-          ERROR {status}: {statusMessage}
-        </Typography>
-        <Typography variant="body1" className={classes.subtitle}>
-          {additionalInfo}
-        </Typography>
-        <Typography variant="h2" className={classes.title}>
-          Looks like someone dropped the mic!
-        </Typography>
-        <Typography variant="h6">
-          <Link to="#" data-testid="go-back-link" onClick={() => navigate(-1)}>
-            Go back
-          </Link>
-          ... or please{' '}
-          <Link to={supportUrl || support.url}>contact support</Link> if you
-          think this is a bug.
-        </Typography>
+    <Box className={classes.parent}>
+      <Grid container className={classes.container}>
+        <Grid item xs={12} sm={8} md={4}>
+          <Typography
+            data-testid="error"
+            variant="body1"
+            className={classes.subtitle}
+          >
+            ERROR {status}: {statusMessage}
+          </Typography>
+          <Typography variant="body1" className={classes.subtitle}>
+            {additionalInfo}
+          </Typography>
+          <Typography variant="h2" className={classes.title}>
+            Looks like someone dropped the mic!
+          </Typography>
+          <Typography variant="h6">
+            <Link
+              to="#"
+              data-testid="go-back-link"
+              onClick={() => navigate(-1)}
+            >
+              Go back
+            </Link>
+            ... or please{' '}
+            <Link to={supportUrl || support.url}>contact support</Link> if you
+            think this is a bug.
+          </Typography>
+        </Grid>
+        <MicDrop />
       </Grid>
-      <MicDrop />
-    </Grid>
+      {stack && <LogViewer text={stack} />}
```

Comment:
Perhaps we could hide this behind a "show details" or something like that, which we always show on the page? Feeling it could be something that everyone has enabled if it's sufficiently out of the way for users that aren't interested

Reply:
<img width="1792" alt="Screenshot 2024-01-11 at 4 02 13 PM" src="https://github.com/backstage/backstage/assets/67240543/2bac0cf9-d642-4440-bb86-0ab75cb8beec">

Modified the page to have an accordion and updated the existing refs to have the stack included by default

Reply:
<img width="1792" alt="295728127-e6e4b3c6-0d25-4b0b-b127-e0c546828489" src="https://github.com/backstage/backstage/assets/67169551/95eebaa7-94ba-47e2-ad29-9bcbab28bf3c">

Why can't it be a very simple `CodeSnippet` right where the red box is in the image?

Reply:
https://backstage.io/storybook/?path=/story/data-display-codesnippet--default

It should just nicely blend in but so that it's there if you need it but not really in your face

Reply:
I agree, here's the updated view. I expanded the content slightly outwards to reasonably fit a stack trace.

<img width="1789" alt="Screenshot 2024-01-11 at 11 01 33 PM" src="https://github.com/backstage/backstage/assets/67240543/3d0d69dc-0e49-49d3-a8b2-cc3460b80d38">

Reply:
Looking good just have another adjustment, can we change the title of this box to just be "Stack Trace" and just the default color, not red. You can then also removed the "Stack Trace" from inside this box/container. It also should be collapsed by default. With those changes I think we'll be done from my perspective 👍

Reply:
Sure thing, here are what the updated changes look like.

<img width="1792" alt="Screenshot 2024-01-12 at 3 21 45 PM" src="https://github.com/backstage/backstage/assets/67240543/4ebf8da9-52c0-40e4-90f1-b0b68e08db6d">

Reply:
Yes! This is exactly what I had in mind 🚀